comixed-webui/src/assets/i18n/fr/app.json

```diff
@@ -111,7 +111,7 @@
       "import-comics": "Bandes Dessinées importées",
       "location-collection": "Lieux",
       "publisher-collection": "Éditeurs",
-      "reading-lists": "Reading Lists",
```

Comment:
Line 58, "title", should be translated.
Line 65, "root", should be translated.

Reply:
The terms Home and Logging were difficult to translate. Home can still be translated as "Accueil", I hadn't thought of that. On the other hand Logging, the translation would be "Journalisation" but I think I have never heard it.We always talk about "log", "logging system". But Quebecers should find it more natural ;)

Reply:
The term I often use and which seems to me the most adapted is logging levels (levels of logs in French).  But it is also often used logging as you can see here: https://www.jmdoudoux.fr/java/dej/chap-logging.htm
So in my opinion logging sounds good or logging levels.